allure-report-face/src/components/widget-overview/OverviewWidget.js

```diff
@@ -1,7 +1,53 @@
 import './styles.css';
 import {ItemView} from 'backbone.marionette';
 import template from './OverviewWidget.hbs';
+import c3 from 'c3';
```

Comment:
Also now I see, that there is no big difference between pie chart on graph tab and this donut. So, you can pie chart into donut with little changes and make them share the same code. Even more, you can place here the same pie chart to keep them consistent.